app/components/Resources/Map.js

```diff
@@ -17,17 +19,30 @@ class Gmap extends Component {
     }
   }
 
-  populateMarkers(markers, map) {
-    let offset = this.props.markers.center ? 2 : 1;
-    markers.forEach((marker, index) => {
-      var latLang = new google.maps.LatLng(marker[0], marker[1]);
-      new google.maps.Marker({
+  setMapOnAll(map, markers) {
+    for (var i = 0; i < markers.length; i++) {
+      markers[i].setMap(map);
+    }
+  }
+
+  generateMarkers(locs, map) {
+    let markers = [];
+    let offset = 1;
+
+    locs.forEach((loc, index) => {
+      var latLang = new google.maps.LatLng(loc[0], loc[1]);
```

Comment:
Talked about code style when it comes to using potentially unclear object indices to get/store variables.  I infer that loc[0] is probably latitude and loc[1] is probably longitude but it can become difficult to sort that out when those situations get more complex.

I'm a fan of being very explicit about what variables are and defining them, but I'm not familiar with standards of javascript as far as that goes.